server/websocket/src/connection.rs

```diff
@@ -208,7 +208,7 @@ where
 
             match msg {
                 // Always accept connection init requests
-                ConnectionInit { payload: _ } => send_message(&msg_sink, ConnectionAck),
+                ConnectionInit { .. } => send_message(&msg_sink, ConnectionAck),
```

Comment:
I don't like this one, I prefer to know what is being ignored. And in case the struct changes, to explicitly ignore (or not) any new fields.

Reply:
Ok, I'll revert it. Please note that the possibility of change (and subsequent requirement for extension of the pattern) is the exact reason we have this lint. The rationale is that as we don't need to know the struct layout to decide here, we can omit it to concentrate on the enum discriminant.

Reply:
That makes sense, I guess you can switch it back if you agree with this lint. Btw you still need to rustfmt with Rust 1.30 so this will pass CI.